<?xml version="1.0"?>
<launch>
    <!-- Camera -->
  <param name="camera_description" command="xacro $(find my_package)/urdf/camera.xacro" />
  <!-- Table -->
  <param name="table_description" command="xacro $(find my_package)/urdf/table.xacro" />

  <!-- push robot_description to factory and spawn robot in gazebo at the origin, change x,y,z arguments to spawn in a different position -->
  <arg name="world_pose" value="-x 0 -y 0 -z 0" />

  <node name="spawn_table_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param table_description -model table $(arg world_pose)"
    respawn="false" output="screen" />

  <node name="spawn_camera_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param camera_description -model camera $(arg world_pose)"
    respawn="false" output="screen" />


</launch>